macros/sql_gen/gen_joined_metrics_cte.sql

```diff
@@ -61,7 +61,7 @@
         ) as {{dim}}
     {%- endfor %}
     {% for metric_name in metric_tree.parent_set %}
-        , nullif({{metric_name}},0) as {{metric_name}}
+        , {{metric_name}} as {{metric_name}}
```

Comment:
This is because we're returning all nulls and making them the BI layer's problem, right?

Reply:
this was to make sure we weren't messing up non-division expression metrics (the original motivation here). If i had `metric_1` with a value of zero, and `metric_2` was `metric_1 + 1`, with this nullif I'd get null instead of 1